src/components/blockquote/blockquote.twig

```diff
@@ -32,6 +32,6 @@
       {% if block_quote__attribution %}
         <figcaption {{ bem('attribution', [], block_quote__base_class) }}>—{{ block_quote__attribution }}</figcaption>
```

Comment:
@mariannuar We should change `figcaption` to `cite`. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite.

Reply:
@amazingrando Changed!

Reply:
@amazingrando Changed!